libvast/src/system/node.cpp

```diff
@@ -195,7 +195,7 @@ spawn_component(const invocation& inv, spawn_arguments& args) {
   auto i = node_state::component_factory.find(inv.full_name);
   if (i == node_state::component_factory.end())
     return make_error(ec::unspecified, "invalid spawn component");
-  VAST_VERBOSE(__func__, VAST_ARG(inv.full_name));
+  VAST_VERBOSE_ANON("node spawns the", inv.name(), "component");
```

Comment:
Wouldn't it be nicer if the message read `node spawns an exporter`?

Reply:
If you can find a better phrasing that works for all of these I'm all ears.

```
2020-09-22T10:09:15.142 node spawns the "accountant" component
2020-09-22T10:09:15.143 node spawns the "type-registry" component
2020-09-22T10:09:15.143 node spawns the "archive" component
2020-09-22T10:09:15.144 node spawns the "index" component
2020-09-22T10:09:15.145 node spawns the "importer" component
```

Reply:
Remove this log message and add component specific ones in the `spawn_*` functions. The messages for singleton components should be `node spawns the accountant`. For non-singleton components you can write `node spawns an exporter for <query>`.

Reply:
Done. Please check if this matches your expectations.